runtime/capsules/src/mctp/base_protocol.rs

```diff
@@ -58,31 +65,39 @@ impl MCTPHeader<[u8; MCTP_HDR_SIZE]> {
         self.set_tag_owner(tag_owner);
         self.set_msg_tag(msg_tag);
     }
+
+    pub fn next_pkt_seq(&self) -> u8 {
+        (self.pkt_seq() + 1) % 4
+    }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone, Copy)]
 pub enum MessageType {
-    MCTPControl,
-    PLDM,
-    SPDM,
-    SSPDM,
-    VendorDefinedPCI,
+    MctpControl = 0,
+    Pldm = 1,
+    Spdm = 5,
+    SecureSPDM = 6,
```

Comment:
nit: We're being a bit inconsistent with if we are capitalizing initialisms `SPDM` or `Spdm`.

I think Tock in general uses `Spdm` style, so we should stick with that.

Reply:
Noted! will change accordingly.